### ISSUE TITLE
[enhancement]: SD Automatic1111 webui extensions access

### ISSUE THREAD

**Issue by metaltiger775** [invoke-ai/InvokeAI#5043]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

I think what frustrates some people about InvokeAI, is that you can't add popular extensions from the Automatic1111 webui, mainly Deforum, to this UI. I think it would be very nice to add support to them since it will allow for way more opportunities, and, at the very least, maybe you guys could manually add like specific extensions that are very popular but idk. I'm just putting an idea out there.

### Alternatives

The only alternative i've considered is just using the webui by Automatic1111, but, honestly, I prefer the UI from InvokeAI, although, for some specific things, I would rather not use it.

**Comment by hipsterusername** [comment]
Unfortunately, it's not quite as simple as that. 

These are incompatible code bases - Auto1111 is older code, and most of the extensions aren't integrated. 

Invoke is based on Diffusers, and much of the core pipelines are built differently.

Extension-builders could easily port over the core capabilities (to varying degrees of difficulty), and create custom nodes. The biggest reason most don't is that most extension devs don't really know how to build the front-end code to support it (hence the use of gradio), but that can easily be solved by working w/ us to help integrate it into the core UI/UX.

TL;DR - We're happy to support capabilities that folks want in the app through either core dev or custom community nodes, but the people you need to be asking is the extension devs.
